.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 temp/
 npm-debug.log
+.DS_Store
```

Comment:
This is something that should be in your global `.gitignore`, not in each and every project's. Please remove.